src/rgw/jwt-cpp/jwt.h

```diff
@@ -659,6 +706,9 @@ namespace jwt {
 			explicit rs512(const std::string& public_key, const std::string& private_key = "", const std::string& public_key_password = "", const std::string& private_key_password = "")
 				: rsa(public_key, private_key, public_key_password, private_key_password, EVP_sha512, "RS512")
 			{}
+
+			rs512(int number, const std::string& modulus, const std::string& exponent) : rsa(EVP_sha512, modulus, exponent, "RS512")
```

Comment:
I had to add an integer to this constructor to avoid the ambiguous constructor problem, is there a better suggestion for this. This I dont think is elegant.

Reply:
a tag struct with a good name, maybe? like,
```c++
struct i_am_a_good_name_t {};
rs512(const i_am_a_good_name_t&, int number, const std::string& modulus, const std::string& exponent) 
 : // ...
{}